php/libraries/User.class.inc

```diff
@@ -435,21 +435,47 @@ class User extends UserPermissions
         /* This is an array of information related to the user. This is used by
          * Zxcvbn to check if a user is doing something foolish like making
          * their password 'johnnyloris'.
-         * We should add words as we think of them.
+         * $lorisKeywords comes from a list generated by the tool 'CeWL'
+         * https://tools.kali.org/password-attacks/cewl
+         * It represents words scraped from the loris.ca and mcin.ca websites
+         * and contains words that users are likely to include in their
+         * passwords.
```

Comment:
I think it's a stretch to go from "this is a list of words used on the loris.ca or mcin.ca website" to "These are likely to be used by users of LORIS who have likely never been to either of those websites". Many of them are just references to projects which aren't likely to be used by anyone who isn't a member of that project, many of them are people's names (but not the logged in user), many of them are complex/scientific words used in texts of press releases which would probably be fairly secure to work into a password.

Just move "loris", "brain", and "mcgill" into a file and then you can include instructions somewhere on how to populate it with their own blacklist data source.